checklists/alz_checklist.en.json

```diff
@@ -3,7 +3,7 @@
         {
             "category": "Azure Billing and Active Directory Tenant",
             "subcategory": "Azure AD Tenants",
-            "text": "How Many Azure AD Tenants do you manage?",
+            "text": "Unless you have a requirement for multitenancy, having more than one Entra tenant will add complexity to architectures and operations",
```

Comment:
This is written more as a suggestion/information, than as a full check.

How would the following work:

Use one Entra tenant for managing your Azure resources, unless you have a clear regulatory or business requirement for multi-tenants.

Reply:
We should also link to https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/landing-zone/design-area/multi-tenant/considerations-recommendations or https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/landing-zone/design-area/multi-tenant/overview

Reply:
Updated based on feedback. Thanks @brsteph.